Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -511,4 +511,9 @@ final class AttributeTests: XCTestCase {
     AssertParse("func takeRepeater(_ f: @MainActor @Sendable @escaping () -> Int) {}")
     AssertParse("takeRepesater { @MainActor @Sendable () -> Int in 0 }")
   }
+
+  func testPackageAttribute() {
+    AssertParse(#"@_package(path: "../my-package") import A"#)
+    AssertParse(#"@_package(url: "https://example.com/package.git", from: "0.0.1") @_exported import A"#)
```

Comment:
Do you think you could also add a few test cases for invalid Swift code? Ideas that come to my mind are 
- misspelled argument labels
- missing commas or colons
- Using identifiers instead of string literals

Reply:
I didn't care of diagnostics for now, so some false syntax may be accepted by the current implementation. The error messages also seemed not helpful at all...

Anyway I'll do that some time. Thanks for your suggestion:)

Reply:
I adds some more valid test cases and I believe the parsing is robust enough now. It's time to dive into invalid cases...